packages/styles/scss/components/dotcom-shell/_dotcom-shell.scss

```diff
@@ -5,22 +5,39 @@
 // LICENSE file in the root directory of this source tree.
 //
 @import '../../globals/imports';
+@import '../footer/index';
+@import '../masthead/masthead';
+@import '../masthead/masthead-l1';
 
 @mixin dotcom-shell {
   .#{$prefix}--dotcom-shell {
     display: flex;
     flex-direction: column;
     min-height: 100vh;
+    padding-top: $layout-07;
+    padding-bottom: $layout-06;
 
     &__content {
       flex: 1;
     }
 
-    &__masthead {
-      width: 100%;
-      position: static;
-      top: 0;
-      z-index: 99999;
+    // Dotcom shell - grid overrides
+    .#{$prefix}--content {
```

Comment:
I don't think we need these override classes. I feel like the content section should be a free game, and adopters can use the grid classes to structure their page.

I need to run to a meeting,  but I'll see if I can come back and provide a little better suggestion here.

Reply:
Same as https://github.com/carbon-design-system/ibm-dotcom-library/pull/97#discussion_r327879567